experiments/golden-results/SPARK-tetris-summary.txt

```diff
@@ -3,6 +3,11 @@ Calling function: Process_Declaration
 Error message: Package declaration
 Nkind: N_Package_Declaration
 --
+Occurs: 11 times
+Calling function: Do_Type_Reference
+Error message: Unsupported declaration of unknown type
```

Comment:
I'm confused by this PR. In the first commit, it's supposedly adding the type to the symtab, and then performs a sanity check that ... fails here? Why are these hit? As far as I understood it this shouldn't happen, unless I've misunderstood the code in the first commit. Was the error message supposed to be hit?

Reply:
The first commit only declares implicit types. This report catches the case where the missing type was not implicit.